src/js/themes/base.d.ts

```diff
@@ -177,12 +177,13 @@ interface ButtonType {
   primary?: ButtonKindType;
   secondary?: ButtonKindType;
   option?: ButtonKindType;
-  active?: ButtonKindType & {
-    default?: ButtonKindType;
-    primary?: ButtonKindType;
-    secondary?: ButtonKindType;
-    [key: string]: ButtonKindType;
-  };
+  active?:
+    | (ButtonKindType & {
```

Comment:
How is this OR affecting the type evaluation? It feels out of place.
```suggestion
    (ButtonKindType & {
```

Reply:
good catch I did not mean to add that OR there